lua/entities/gmod_wire_expression2/core/custom/tardis2.lua

```diff
@@ -16,9 +16,13 @@ local function getTardis(ent)
     end
 end
 
-local function HandleE2(ent, name, ...)
+local function HandleE2(owner, ent, name, ...)
```

Comment:
It would be good to rearrange this so that `self` was passed in as the first argument and then we could grab the `player` off that as well as pass it into `ent:HandleE2(name, self, ...)` like so, rather than passing in `self.player` and then `self` later on in the same call.

Reply:
On further thinking actually I think it might be best left to the HandleE2 function to deal with this which already has the player passed in as part of the E2 context.

I think the GetX functions are okay to run against any TARDIS regardless of isomorphic for example which we can only really handle in that function. There is an `ENT:CheckSecurity(ply)` function you can use to check permissions for this for things like `SetFlight` etc

Reply:
Looks good!